### PR TITLE
✨ dashboard authorization UI updates

### PR DESCRIPTION
**Pull request by sreehari-aot**
# Issue Tracking

JIRA: 
Issue Type: FEATURE
https://aottech.atlassian.net/browse/FWF-1638

# Changes

- Updated the Dashboard component to use New authorization endpoints
- Removed unused code 
- Updated test cases
- Updated style to have min height for the admin panel
- Insights updated to use new authorization API

# How has the change been tested?

Unit tests, Integration tests, Manual verification, etc.

### CODE REVIEW

File: forms-flow-web/src/apiManager/services/dashboardsService.js [hunk 1]
```diff
@@ -75,70 +50,44 @@ export const fetchGroups = () => {
...
-  let newGroups = [...groups];
-
-  for (let group of newGroups) {
-    group.dashboards = getCleanedDashboards(group.dashboards);
-  }
-
-  return newGroups;
+export const updateAuthorization = (data) => {
+  return (dispatch) => {
+    httpPOSTRequest(API.DASHBOARD_AUTHORIZATION, data)
```
Comment (sumesh-aot) [inline forms-flow-web/src/apiManager/services/dashboardsService.js:55]: Not specific to this change; but can't we handle the timeout globally? Like wrap the http request and add time out there ?

**Review by sumesh-aot — APPROVED** [review]

File: forms-flow-web/src/apiManager/services/dashboardsService.js (hunk 1 above)
Comment (sreehari-aot) [reply forms-flow-web/src/apiManager/services/dashboardsService.js:55]: The timeout looks hacky to me, will replace it with toast

**Review by abhilash-aot — APPROVED** [review]

**Review by abhilash-aot — COMMENTED** [review]
Is there any update or migration for existing users using the dashboard with groups? @sreehari-aot / @sumesh-aot

**Comment by sreehari-aot** [comment]
> Is there any update or migration for existing users using the dashboard with groups? @sreehari-aot / @sumesh-aot

If there are existing users they have to migrate the authorizations manually

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**